packages/boot/test/bootstrapTests/ec-membership-update.test.ts

```diff
@@ -6,13 +6,15 @@ import {
   slotToBoardRemote,
   unmarshalFromVstorage,
 } from '@agoric/vats/tools/board-utils.js';
-import { makeMarshal } from '@endo/marshal';
+import { makeMarshal, passStyleOf } from '@endo/marshal';
 
 import { makeSwingsetTestKit } from '../../tools/supports.js';
 import {
   makeGovernanceDriver,
   makeWalletFactoryDriver,
 } from '../../tools/drivers.js';
+import { makeLiquidationTestKit } from '../../tools/liquidation.js';
+import { NonNullish } from '@agoric/internal';
```

Comment:
Does this pass lint? don't all the intra-package imports (`../`) have to go _after_ all the inter-package imports?

Try optimize imports?

_not critical, unless our lint rules require it_

Reply:
its required. for some reason the lint issues are not actively appearing in my IDE. 
thanks for the catch